diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index e9fc367..d9ff2dc 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -3,7 +3,7 @@ const Pdf = require('./lib/pdf-parse.js');
 
 module.exports = Pdf;
 
-let isDebugMode = !module.parent; 
+let isDebugMode = false; 
 
 //process.env.AUTO_KENT_DEBUG
 
@@ -11,6 +11,8 @@ let isDebugMode = !module.parent;
 //for testing purpose
 if (isDebugMode) {
 
+    console.log("Hola, estoy ignorando el false");
+
     let PDF_FILE = './test/data/05-versions-space.pdf';
     let dataBuffer = Fs.readFileSync(PDF_FILE);
     Pdf(dataBuffer).then(function(data) {
